src/Validator/Range.php

```diff
@@ -130,7 +130,7 @@ public function isValid(mixed $value): bool
                 break;
             case self::TYPE_FLOAT:
                 $value = $value + 0;
-                if(!is_float($value) && !is_int($value)) {
+                if(!is_float($value) && is_int($value)) {
```

Comment:
Is this supposed to be changed?

Reply:
PhpStan complained that the condition put there will always return `false`. So, this change was done. We can revert this back and have to explicitly add a line for PhpStan to ignore this condition then. Let me know how you want me to proceed with this.

Reply:
@TorstenDittmann, do you think it's safe to remove the check? I think `$value` always be a float or int after `$value = $value + 0` because either it will be cast to a number or an exception is thrown.

Reply:
Okay, did some testing now. This should work as expected:
```php
$value = $value + 0.0;
if (! is_numeric($value)) {
    return false;
}
```

Reply:
@TorstenDittmann when would `$value` not be numeric after `$value = $value + 0.0`?

Reply:
@TorstenDittmann, perhaps this block should be:

```php
            case self::TYPE_FLOAT:
                if (! is_numeric($value) ) {
                    return false;
                }
                $value = $value + 0;
                break;
```

Reply:
```php
            case self::TYPE_FLOAT:
                if (! is_numeric($value) ) {
                    return false;
                }
                $value = $value + 0.0;
                break;
```

We can agree on this 👍🏻

Reply:
Addressed this in the PR.